lib/pinyin/parsers.ex

```diff
@@ -58,6 +58,14 @@ defmodule Pinyin.Parsers do
     ê o yo n ng wong m
   )
 
+  # Standalone finals that look like initial and final
+  # Must be parsed as final, otherwise conversion to zhuyin won't be correct.
+  # For example 吃 is chi where the i is not a final.
+  surprising_standalone_finals = ~w(
```

Comment:
These finals here behave very similarly as the other standalone finals. The difference is that the entries in the already existing list of standalone finals are unique. For example "wang" - if we didn't have the standalone finals list, it wouldn't be able to be parsed. So it is always a standalone final, never a combination.
But "chi" is a standalone final by meaning but the parser would incorrectly classify it as an initial ch+i.

It's not just an artifact of conversion to Zhuyin. If you try to pronounce "chi" you see that the "i" is not a final. It's not pronounced like the "i" sound (which confusingly is spelled as "yi" when used as a standalone final). Where the "i" in "qi" has the "i"/"yi" sound.

That's one reason why I like to know Zhuyin and Pinyin. Makes me understand better how the sounds are composed.

Reply:
Okay. Perhaps we could also refer to the bopomofo wikipedia page here so that we have some reference we can look at if we ever need to adjust the parser again.

Besides that, let's hope we don't find some uncommon character in a year or so where "chi" _is_ an initial + final.